contrib/README.md

```diff
@@ -3,7 +3,6 @@ Repository Tools
 
 ### [Developer tools](/contrib/devtools) ###
 Specific tools for developers working on this repository.
-Contains the script `github-merge.py` for merging github pull requests securely and signing them using GPG.
 
 ### [Verify-Commits](/contrib/verify-commits) ###
```

Comment:
I assume verify-commits stays for convenience?

Reply:
AFAIK we don't have a verify-commits in a separate repo; the one for the main repo is also in its contrib.

Reply:
I meant that it's a copy of the same script? Idk nvm